src/main/java/org/folio/service/locale/LocaleSettingsServiceImpl.java

```diff
@@ -0,0 +1,111 @@
+package org.folio.service.locale;
+
+import java.util.Currency;
+import java.util.Locale;
+import java.util.Optional;
+import java.util.concurrent.CompletableFuture;
+import java.util.function.BiConsumer;
+
+import io.vertx.core.buffer.Buffer;
+import io.vertx.core.http.HttpClientResponse;
+import io.vertx.core.json.JsonObject;
+import io.vertx.core.logging.Logger;
+import io.vertx.core.logging.LoggerFactory;
+import org.jetbrains.annotations.NotNull;
+import org.joda.time.DateTimeZone;
+import org.springframework.beans.factory.annotation.Autowired;
+
+import org.folio.holdingsiq.model.OkapiData;
+import org.folio.holdingsiq.service.impl.ConfigurationClientProvider;
+import org.folio.rest.client.ConfigurationsClient;
+import org.folio.rest.tools.client.Response;
+import org.folio.rest.tools.utils.TenantTool;
+
+public class LocaleSettingsServiceImpl implements LocaleSettingsService {
+
+  private static final Logger LOG = LoggerFactory.getLogger(LocaleSettingsServiceImpl.class);
+  private final String query = "module=ORG and configName=localeSettings";
+
+  private ConfigurationClientProvider configurationClientProvider;
+
+  @Autowired
```

Comment:
`@Autowired` could be removed because bean creation is declared in `AppConfig`

Reply:
removed